src/lib/components/workflow/workflow-advanced-search.svelte

```diff
@@ -87,6 +95,12 @@
       >
         {translate('common.search')}
       </Button>
+      <CopyButton
```

Comment:
We should only need this `CopyButton` in `src/lib/components/search-attribute-filter/index.svelte` since `SearchAttributeFilter` is the parent component of `WorkflowAdvancedSearch`, but let me know if you run into issues with that approach.
We should just need to remove the `showFilter` conditional so it also shows when the raw query is shown and maybe fix some styling. 

We might also want to consider adding a tooltip (e.g. "Copy Raw Query") so users know what the button does.